assets/js/redux/utils/makeError.js

```diff
@@ -1,6 +1,7 @@
-export default function makeError(severity, text) {
+export default function makeError(severity, text, id) {
```

Comment:
unused parameter? id is passed but the only function call the makeError doesn't utilize this extra param

https://github.com/Code4HR/okcandidate/blob/b4dba2469b3d706b049847e84300db73b5b5d11d/assets/js/redux/survey/survey-reducer.js#L189